### PR TITLE
feat: add tests and ci

### PR DESCRIPTION
**Pull request by mcdurdin**
Starts a basic link check CI action.

### CODE REVIEW

**Comment by darcywong00** [comment]
Re: the failing tests (broken links). It looks like landing pages are ok (burmese), but many of the others depend on keyboard search results?

File: _includes/includes/KeymanHosts.php [hunk 1]
```diff
@@ -7,9 +7,11 @@ class KeymanHosts {
     // Three tiers: development = x.keyman.com.local
     // Staging = staging.x.keyman.com
     // Production = x.keyman.com
+    // Test = GitHub actions
```
Comment (darcywong00) [inline _includes/includes/KeymanHosts.php:10]: l. 7 now Four tiers?

File: _includes/includes/KeymanHosts.php [hunk 2]
```diff
@@ -7,9 +7,11 @@ class KeymanHosts {
     // Three tiers: development = x.keyman.com.local
```
Comment (darcywong00) [inline _includes/includes/KeymanHosts.php:7]:
```suggestion
    // Four tiers: development = x.keyman.com.local
```

File: testimonials/index.php [hunk 3]
```diff
@@ -1,29 +1,29 @@
...
-  ]);           
+  ]);
 ?>
 <h2 class="red underline">What people are saying about Keyman Desktop</h2>
 
 <p class="center">
   <span class="testimonial">"... I would suggest to stick with keyman and I am sure you would love the experience.<br/>
   I type #Amharic like a Pro on Keyman [winking face]"</span>
-  <h4 class="caption"><a href="https://twitter.com/MekonnenAkele/status/1208281934721409025">@MekonnenAkele</a> - December 2019</h4>
+  <h4 class="caption"><a href="https://twitter.com/MekonnenAkele/status/1208281934721409025" target="_blank">@MekonnenAkele</a> - December 2019</h4>
```
Comment (darcywong00) [inline testimonials/index.php:16]: I thought previously you wanted to get away from having `target="_blank"` in the links.

Or is it to use for external links?

Comment (mcdurdin) [reply testimonials/index.php:16]: For these external links, I am okay with `_blank`. Most places, though, nope :)

File: testimonials/index.php [hunk 4]
```diff
@@ -102,7 +102,7 @@
 </p>
 <p class="center">
   <span class="testimonial">"We are delighted to be in a position to distribute Aboriginal language keyboard solutions to our clients. The local Saanich tribe will be the first to benefit ... You can be sure that your invention will be a wonderful enabling tool for a host of new Indigenous communities via their connections to First Peoples' Cultural Foundation."</span>
-  <h4 class="caption">Peter Brand - <a href="www.fpcf.ca" target="blank">First Peoples' Cultural Foundation</a>, British Columbia</h4>
+  <h4 class="caption">Peter Brand - <a href="https://www.fpcf.ca/" target="_blank">First Peoples' Cultural Foundation</a>, British Columbia</h4>
```
Comment (darcywong00) [inline testimonials/index.php:105]: Maybe their site is http only? On my Win 10 / Chrome
https://www.fpcf.ca/ times out
http://fpcf.ca/ gets served up

**Pull request merged**